tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<p>It's actually not a 2π-pulse since the area is not 2π. I guess you can just omit calling it that, maybe reference Rabi oscillations instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Indeed, don't know why I said that</p>